spot_rl_experiments/configs/config.yaml

```diff
@@ -135,8 +137,10 @@ INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE: [0, -180, 180, 0, 0, 90] # The initial
 INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE_LEFT_HAND: [0, -180, 180, 0, 0, -90] # The initial orientation of the arm for side grasping (gripper is in object's left hand side)
 INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE_TOP_DOWN: [-0.91, -92.87, 91.21, 0, 90.01, 0] # The initial orientation of the arm for top down grasping
 # The old gaze ready angle: [0, -170, 120, 0, 75, 0]
-GAZE_ARM_JOINT_ANGLES: [0, -160, 100, 0, 75, 0] #[0, -160, 100, 0, 90, 0]
-SEMANTIC_PLACE_ARM_JOINT_ANGLES: [0, -125, 80, 0, 85, 0]
+GAZE_ARM_JOINT_ANGLES: [0, -160, 100, 0, 75, 0]
+GAZE_ARM_JOINT_ANGLES_LOW_RECEPTACLES: [0, -160, 100, 0, 75, 0] #[0, -160, 100, 0, 90, 0]
+GAZE_ARM_JOINT_ANGLES_HIGH_RECEPTACLES: [0, -120, 60, 0, 88, 0] #[0, -160, 100, 0, 90, 0]
+HEIGHT_THRESHOLD : 0.8128
```

Comment:
why this value?

Reply:
Hi Jimmy, myself and @joannetruong did teleop of the arm in the lab, and noted these values - the low receptacle being the old value (just the gripper now pointing down looking on the receptacle). The higher receptacle values were noted based on how the gripper camera viewed the table of 42 inch and higher.

Reply:
For the Height_threshold, we decided to keep it as 32 inch table rn - as when we tested in the lab, objects above 32 inch in our setup [ie 42 in and 52in table setup] required higher gazing angle.